pages/careers/_open-positions.html

```diff
@@ -237,5 +320,267 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
+        You'll be responsible for setting the direction for this library, which is written in Terraform, Go, Bash, and
+        Python and used in production by hundreds of companies.
+      </li>
+      <li>
+        <strong>Reference Architecture.</strong>
+        You'll also be the owner of the
+        <a href="/reference-architecture" target="_blank">Reference Architecture</a>, an opinionated, end-to-end tech
+        stack built on top of the Infrastructure as Code Library that we can deploy into a customer's AWS accounts in
+        about one day. You'll be responsible for defining the next generation of the Reference Architecture as part of
+        a major upcoming redesign.
+      </li>
+      <li>
+        <strong>Compliance.</strong>
+        Finally, you'll also be the owner of
+        <a href="achieve-compliance" target="_blank">Gruntwork Compliance</a>, which includes out-of-the-box compliance
+        with the CIS AWS Foundations Benchmark today, and soon, SOC 2. You'll set the direction for our compliance
+        business, including figuring out which standards we'll target in the future (e.g., ISO 27001, PCI, HIPAA, other
+        CIS benchmarks, etc.), how to package the product, how to help customers ensure ongoing compliance, and so on.
+      </li>
+      <li>
+        <strong>Work alongside engineering and design.</strong>
+        You'll serve as one member of an executive "trio" that includes an engineering manager and product designer.
+        You'll collaborate with your colleagues on the vision, usability, and business viability of this new product,
+        and take primary responsibility for our ability to execute. Further, you'll work directly with and report to
+        one of our co-founders.
+      </li>
+      <li>
+        <strong>Work alongside customers.</strong>
+        You'll work directly with our customers to understand what problems they are facing, figure out solutions to
+        those problems, guide the team in building those solutions, share those solutions with customers, gather
+        feedback, and iterate.
+      </li>
+      <li>
+        <strong>Drive our product management practices.</strong>
+        Help improve how we build products as a company by streamlining all of our product management processes,
+        including product planning, user testing, user story mapping, task tracking, backlog grooming, sprint planning
+        and execution, market research, pricing strategy, customer communication, and more.
+      </li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong>
+        As the future owner of several major new products at Gruntwork, we'll look to you for leadership, insights, and
+        methodologies that will help us translate our shared vision into a fantastic experience for customers.
+      </li>
```

Comment:
```suggestion
      <li>
        <strong>Product leadership.</strong>
        You have several years of experience in product management to give both us and you confidence that you can bring leadership, insights, and
        methodologies that will help us translate our shared vision into a fantastic experience for customers.
      </li>
```

Reply:
Actually, maybe this isn't quite right. Perhaps what we want to emphasize here is specific skill and experience with *product vision.* That is, the person has experience taking in large amounts of information and synthesizing that into a coherent and compelling product vision. I know it's trite, but I think we want proven experience here.